buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1640,7 +1640,7 @@ class BeamModulePlugin implements Plugin<Project> {
             executable 'sh'
             args '-c', ". ${project.ext.envdir}/bin/activate && cd ${copiedSrcRoot}/sdks/python && python setup.py sdist --formats zip,gztar --dist-dir ${project.buildDir}"
           }
-          def collection = project.fileTree("${project.buildDir}"){ include '**/*.tar.gz' exclude '**/apache-beam.tar.gz'}
+          def collection = project.fileTree("${project.buildDir}"){ include '**/*.tar.gz' exclude '**/apache-beam.tar.gz', 'srcs/**'}
```

Comment:
I didn't see the problem here. [Log](https://scans.gradle.com/s/l7em7nt3nuxaq/console-log?task=:beam-sdks-python-test-suites-dataflow:sdist#L1193) shows it's working well. Can you add your failure or explain more?

Reply:
I get it when I try run "naked" `sdist` task (`./gradlew --stacktrace beam:sdks:python:sdist`)
```
Caused by: java.lang.IllegalStateException: Expected directory '/Users/kasia/Repos/beam/sdks/python/build' to contain exactly one file, however, it contains more than one file.
        at org.gradle.api.internal.file.AbstractFileCollection.getSingleFile(AbstractFileCollection.java:65)
        at org.apache.beam.gradle.BeamModulePlugin$_apply_closure20$_closure136$_closure145.doCall(BeamModulePlugin.groovy:1644)
```

Reply:
Thanks! For some reason I can't reproduce it on my machine, but I saw same complain in dev list. I'll include your fix.